src/test.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+from integration.gen_1_pokemon import Pokemon
+
+class TestPokemonDisplay(unittest.TestCase):
```

Comment:
tbh it's nice to have tests here, we slacked on them pretty hard in initial setup, thanks!